### ISSUE TITLE
PR approved by whom avatars

### ISSUE THREAD

**Issue by foodpoison** [sindresorhus/refined-github#812]
Hi guys, good job on the plugin.
I have a small request, similarly to how we currently have the Reaction Avatars, it would be great to show avatars of people that have so far approved the PR.
Our team goes by a "two approval before you can merge" policy, and this will greatly speed up that process, at a glance we can immediately see which PRs are ready to merge, as opposed to having to click into the PR to count the number of clicks.

**Comment by bfred-it** [comment]
It's already there, provided by GH in the sidebar:

<img width="224" alt="" src="https://user-images.githubusercontent.com/1402241/32774763-4e953292-c968-11e7-9d47-93a670ca011b.png">

**Comment by hkdobrev** [comment]
>  at a glance we can immediately see which PRs are ready to merge, as opposed to having to click into the PR to count the number of clicks.

@bfred-it I think what @foodpoison means is that they want avatars in the PR list next to the green check. At the moment the GitHub filter `review:approved` considers a PR as approved as soon as one person approves, you cannot filter by a number of approvals.

**Comment by bfred-it** [comment]
Oh, got it, I had missed the last sentence.

From:

<img width="358" alt="Approved" src="https://user-images.githubusercontent.com/1402241/32776318-1f5f4a58-c96d-11e7-8e57-732d11598dd2.png">

To:

<img width="458" alt="Approved by Tizio and Sempronio" src="https://user-images.githubusercontent.com/1402241/32776322-21ecedd4-c96d-11e7-9285-a00ed1315529.png">

**Comment by bfred-it** [comment]
Showing the numbers appears pretty easy, it already appears in the attributes:

<img width="236" alt="aria label 2 review approvals" src="https://user-images.githubusercontent.com/1402241/32776703-5f2c5d6e-c96e-11e7-8ca9-4d9feaf906a0.png">

It's a good start, PRs welcome!

**Comment by hkdobrev** [comment]
> Showing the numbers seems pretty easy, it already appears in the attributes

@bfred-it Wow, nice catch! I could really use that!

I guess as a start we can just visualise that ARIA label somehow and think about something more later down the road.

The question is how to visualise it - just as a number or visualise it by multiple icons.

---

I've checked the ARIA label on different pull requests and I have some bad news.

Here are some of the different cases:
- "Review required before merging" (when no one has reviewed yet)
- "1 review requesting changes" (even when one person has requested changes multiple times which is completely fine)
- "1 review approval" - works as expected
- "2 review approvals" - works as expected
- "4 reviews requesting changes" - This is shown even after 3 of those reviewers have approved and only one hasn't. I.e. you see this in the PR sidebar: 

<img width="89" alt="screen shot 2017-11-15 at 00 22 10" src="https://user-images.githubusercontent.com/506129/32808250-0f942a2a-c99b-11e7-9fbb-8dd7d722f03b.png">

**Comment by bfred-it** [comment]
@hkdobrev yeah the information isn't great when there are change requests, but if the change requests have been fulfilled they can be dismissed by collaborators.
